proxylib/memcached/parser.go

```diff
@@ -130,7 +136,11 @@ func L7RuleParser(rule *cilium.PortNetworkPolicyRule) []proxylib.L7NetworkPolicy
 			}
 		}
 		if !commandFound {
-			proxylib.ParseError("command not specified", rule)
+			if len(br.keyExact) > 0 || len(br.keyPrefix) > 0 || br.regex != nil {
+				proxylib.ParseError("command not specified but key was provided", rule)
+			} else {
+				br.empty = true
+			}
```

Comment:
While looking at this I noticed potential problem with `commandFound`. Currently it is set regardless the result of the map lookup. Should it be set like this instead?
```
			switch k {
			case "command":
				br.commands, commandFound = MemcacheOpCodeMap[v]
```

Reply:
This would actually panic without commandFound, but we can also do it your way.